src/test/java/org/openrewrite/java/testing/junit5/AddParameterizedTestAnnotationTest.java

```diff
@@ -0,0 +1,116 @@
+package org.openrewrite.java.testing.junit5;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.Issue;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class AddParameterizedTestAnnotationTest implements RewriteTest {
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion()
+            .classpathFromResources(new InMemoryExecutionContext(), "junit-4.13", "junit-jupiter-api-5.9"))
```

Comment:
We're only using JUnit Jupiter here, so there's no need for the JUnit 4 dependency.
```suggestion
            .classpathFromResources(new InMemoryExecutionContext(), "junit-jupiter-api-5.9"))
```